.github/workflows/turborepo-release.yml

```diff
@@ -147,12 +147,16 @@ jobs:
       - name: Setup capnproto
         uses: ./.github/actions/setup-capnproto
 
+      - name: "Get toolchain version from file"
+        id: file
+        shell: bash
+        run: echo "toolchain=$(cat ./rust-toolchain.toml | grep -F 'channel = ' | sed 's/channel = //' | sed 's/\"')" >> $GITHUB_OUTPUT
+
       - name: Rust Setup
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@master
```

Comment:
We cannot use `actions-rust-lang/setup-rust-toolchain` like we do in other places as under the hood it uses `curl --retry-connrefused` which was added in curl 7.52.0, but curl 7.47.0 is the latest version available to old Ubuntu Xenial version we use to compile our Linux binaries.